compiler/circlechef/tests/shape_signature/test.recipe

```diff
@@ -41,5 +41,15 @@ operation {
     activation: NONE
   }
 }
+operation {
```

Comment:
plz do not edit this file, add new test recipe.

Reply:
> plz do not edit this file, add new test recipe.

Do you mean test recipe in res/CircleRecipes/.... and res/TensorFlowLiteRecipes/... ?
Then, I've already added. please ignore this PR. :)

Reply:
> please ignore this PR. :)

OK. FYI `circlechef/tests` are mosting testing of new features of circle chef.